packages/web/src/components/edit/fields/DatePickerField.tsx

```diff
@@ -40,7 +40,19 @@ export const DatePickerField = (props: DatePickerFieldProps) => {
   useEffect(() => setIsFocused(shouldFocus ?? false), [shouldFocus])
 
   return (
-    <>
+    <Flex
```

Comment:
just a thought, we do this a bit, we might want to add hover options to flex/paper

Reply:
I like that